src/components/Hue.tsx

```diff
@@ -1,10 +1,17 @@
 import React, { useCallback } from "react";
+
 import Interactive from "./Interactive";
-import formatClassName from "../utils/formatClassName";
-import hsvToHslString from "../utils/hsvToHslString";
+
 import styles from "../styles.css";
+import { hsvToHslString } from "../utils/convert";
+import formatClassName from "../utils/formatClassName";
+
+interface Props {
+  hue: number;
+  onChange: (newHue: { h: number }) => void;
+}
 
-const Hue = ({ hue, onChange }) => {
+const Hue = ({ hue, onChange }: Props) => {
   const handleMove = useCallback(
     (interaction) => {
```

Comment:
Shouldn't we add `interaction: Interaction` here?

Reply:
In `Hue`'s props?